charts/alluxio/docker/init-users/cache_clean.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -e
+
+rm -rf /dev/shm/*/*/alluxioworker
```

Comment:
`rm -rf /dev/shm/*/*/alluxioworker`  I think this change is very dangerous.

Reply:
1. the cache dir is configurable, it's not `/dev/shm`
2. there are many cache directories which shared the same base directory. So it may delete other dataset's cache directory.

Reply:
@cheyang Yep, cache dir now is configurable, i am sorry for forgetting it.
It is better to delete same namespace and name cache in same node.

Reply:
1. I agree with @cheyang for the first suggestion: cache dir is configurable. Moreover, there may be multiple cache dirs when user specifies multi-dir multi-tiered cache stores( Refer to [here](https://github.com/fluid-cloudnative/fluid/blob/master/docs/zh/samples/tieredstore_config.md#%E5%8D%95%E5%B1%82%E5%A4%9A%E7%9B%AE%E5%BD%95%E5%AD%98%E5%82%A8%E9%85%8D%E7%BD%AE) )
2. For the second one, I don't think the `rm -rf` command can delete other dataset's cache directory becacuse the command will be executed in the init-container where only cache dirs for this dataset is mounted.

Reply:
I agree with @cheyang and @TrafalgarZZZ: 1) The cache dir should be configurable; 2) multi tired dir should be considered.
And I am confused about the 2nd suggestion of @TrafalgarZZZ.
Alluxio will clean orphan blocks  on startup by default (see `alluxio.master.startup.block.integrity.check.enabled` in [Alluxio's Properties list](https://docs.alluxio.io/os/user/stable/en/reference/Properties-List.html)). However, sometimes it won't work. So I think it is more like a bug of Alluxio.

Reply:
@TrafalgarZZZ I have changed the logic of importing env of mounting path now. This env will be imported into the init container even if not to init user.